blocks/api/parser.js

```diff
@@ -225,7 +225,8 @@ export function createBlockWithFallback( name, rawContent, attributes ) {
  */
 export function parseWithGrammar( content ) {
 	return grammarParse( content ).reduce( ( memo, blockNode ) => {
-		const { blockName, rawContent, attrs } = blockNode;
+		// eslint-disable-next-line no-unused-vars
+		const { blockName, innerBlocks, innerHTML: rawContent, attrs } = blockNode;
```

Comment:
We probably want to normalize this toward `innerHTML` and drop `rawContent`, for consistency, and also because objects of `innerBlocks` will keep their `innerHTML` property. Should be fine to do separately.